lib/index.js

```diff
@@ -6,19 +6,21 @@ var cache = new Map();
 
 
 /**
- * Mocks a method of an object.
+ * Mocks a value of an object.
  *
  * @param {Object} obj
  * @param {string} key
- * @param {!Function} method
+ * @param {!Function|Object} value
  */
-var method = module.exports = (obj, key, method) => {
-  method = method === undefined ? () => {} : method;
+var method = module.exports = (obj, key, value) => {
+  var noop = () => {};
+  value = value === undefined ? noop : value;
```

Comment:
Now that I think about it, It's ok if `value` is `undefined`. I think that check was put there before because this library used to be used to mock methods only, then it changed to mocking any value.

Feel free to keep or remove this line

Reply:
I agree that the value should not have default value now